src/i18n/en/propagation_modal.ts

```diff
@@ -19,4 +19,5 @@ export default {
                 If you are still unable to get the correct results, you can check the propagation status using a <a href="https://dnschecker.org/">propagation checker</a>.
     `,
     title: "DNS Propagation",
+    close: "close",
```

Comment:
I don't like that we're duplicating this in https://github.com/do-community/dns-tool/blob/8396e821d5d2ea270024bc11d2abcb61c99e8521/src/i18n/en/record_selection_modal.ts#L8 - I think we should have some sort of central files for common strings?